runtime/src/lib.rs

```diff
@@ -429,6 +434,7 @@ impl pallet_exchange::Config for Runtime {
 }
 
 parameter_types! {
+	// Is this ok?
```

Comment:
@enthusiastmartin

Reply:
the value is ok. I am not sure what the reward amount should be, hence 0 for now.

We can also consider doiing this #122

Reply:
The reward should be bigger/same as the amount paid for the tx but smaller than the dust amount.